servers/dev-tool-setup/main.go

```
@@ -0,0 +1,275 @@
+			// 		{GitlabUsername: "ge63sir"},
+			// 	},
+			// 	Coach:       TeamMember{GitlabUsername: "ge64fef"},
+			// 	ProjectLead: TeamMember{GitlabUsername: "ge35qis"},
+			// },
+		},
+	}
+	_ = gitlabconfig
+	_ = jiraconfig
+	_ = confluenceconfig
+
+	setupGitlab(gitlabconfig, semester)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)